src/components/Avatar.js

```diff
@@ -34,6 +35,13 @@ const defaultProps = {
 };
 
 class Avatar extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isErrorFetchingImage: false,
```

Comment:
`isErrorFetchingImage` can we rename it to `imageError` or `showFallbackAvatar` ?
@jayeshmangwani 
cc: @AndrewGable

Reply:
replaced isErrorFetchingImage with imageError